src/plugins/custom_import_map/public/components/vector_upload_options.tsx

```diff
@@ -164,45 +164,88 @@ const VectorUploadOptions = (props: RegionMapOptionsProps) => {
     }
   };
 
+  const formatSuccessToast = (indexName: string, totalRecords: number) => {
+    const textContent =
+      totalRecords > 1
+        ? 'Successfully added ' + totalRecords + ' features to ' + indexName + '.'
+        : 'Successfully added 1 feature to ' + indexName + '.';
+    notifications.toasts.addSuccess({
+      text: toMountPoint(
+        <div>
+          <p>{textContent} Refresh to visualize the uploaded map.</p>
```

Comment:
in case of error do we still need to refresh and visualize the map? Is this message correct in case of errors

Reply:
Only in case of partial errors, since few features get indexed and not all - so users can still visualize that map. Check the PR description for the toast messages in case of success and partial failure cases : )